packages/orbit-components/src/Wizard/index.jsx

```diff
@@ -12,12 +12,11 @@ import Portal from "../Portal";
 import Modal from "../Modal";
```

Comment:
I noticed one problem in in mobile Wizard, see what happens when you hover over items on mobile, I made the borders red so that you can see it better:

https://user-images.githubusercontent.com/471278/148947274-8787f03c-67b7-4b37-a709-277a3c551631.mov

The background color is going over both the separation lines and over the left border on the active item.